concepts/05 Widgets/Slider/05 Customize Widget Appearance.md

```diff
@@ -14,8 +17,104 @@ The **Slider** can display labels for the [min](/api-reference/10%20UI%20Widgets
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-slider
+        <dxo-label
+            [visible]="true"
+            [format]="format"
+            position="bottom"
+        ></dxo-label>
+    </dx-slider>
+
+    <!--TypeScript-->
+    import { DxSliderModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        format = function(value) {
+            return value + " units";
+        };
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxSliderModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <template>
+        <DxSlider
+            :label="label"
```

Comment:
Use nested components. Please fix this in all these examples.

Reply:
fixed